pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -198,7 +198,7 @@ func (r *Builder) DataVolumes(vmRef ref.Ref, secret *core.Secret, configMap *cor
 			return
 		}
 		for _, da := range vm.DiskAttachments {
-			if da.Disk.StorageDomain == sd.ID {
+			if da.Disk.StorageType != "lun" && da.Disk.StorageDomain == sd.ID {
```

Comment:
ok, later I saw that the critical validation of the storage type prevents proceeding with cinder or managed block storage, but still I think it would make more sense to limit this to "image" instead